File: Code Chef/JavaScript/800-899/809 Complete the credits CREDITS.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => Number(strItem.trim()));
    
    return intArray;
};

const findTypeOfSem = (semCredits) => {
    const typeOfSem = (semCredits > 65)
        ? 'Overload'
        : (semCredits < 35)
            ? 'Underload'
            : 'Normal';
            
    console.log(typeOfSem);
};

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((semCredits) => {
        findTypeOfSem(semCredits);
    });
});

File: Code Chef/C Programming/800-899/809 Complete the credits CREDITS.c
#include <stdio.h>
int main(void)
{
	int t,x;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d",&x);
	    if(x<35)
	    {
	        printf("Underload\n");
	    }
	    else if(x>65)
	    {
	        printf("Overload\n");
	    }
	    else
	    {
	        printf("Normal\n");
	    }
	}
	return 0;
}
